mlir/lib/Dialect/Tosa/IR/TosaOps.cpp

```diff
@@ -1238,6 +1487,11 @@ LogicalResult tosa::TileOp::inferReturnTypeComponents(
 }
 
 LogicalResult tosa::TileOp::verify() {
+  if (verifySameElementTypes(*this, /* intype = */ getInput1().getType(),
```

Comment:
Are all these verifiers part of the fp8 support? It doesn't feel like it. Can we pull into a separate patch?

Reply:
Yes, it doesn't look like. I will put them into other patches.

Reply:
change refactored to this patch: https://github.com/llvm/llvm-project/pull/127923

Reply:
https://github.com/llvm/llvm-project/pull/127923 merged